docs/wiki/NuGet-packages.md

```diff
@@ -13,6 +13,31 @@ This package is primarily intended as a method for rapidly shipping hotfixes to
 Note: this package is intended as a replacement for Microsoft.Net.Compilers (which is a Windows-only package) and Microsoft.NETCore.Compilers. Those packages are now deprecated and will be deleted in the future.
 
 ## Versioning
+Starting on Roslyn versions `5.0` and above, Roslyn NuGet packages are shipped with the .NET SDK instead of VS.  New Roslyn packages will be published on every SDK preview and GA release.
+   
+Prior to this change, a Roslyn package was matched exactly to a VS version, and did not exactly match SDK versions.  Now, Roslyn packages will exactly align with an SDK version, but will no longer exactly match a VS version.  While generally the SDK releases ship alongside new VS versions, there may be drift in the exact commit of Roslyn when compared to VS (especially for prereleases).
+
+### Version Schema
```

Comment:
PTAL at these version changes.  The versions will mostly be the same for stable packages.  Preview packages will have build numbers.  I've tried to explain the logic below but let me know if it needs more info.